src/transformers/utils/generic.py

```diff
@@ -200,17 +232,39 @@ def to_py_obj(obj):
     """
     Convert a TensorFlow tensor, PyTorch tensor, Numpy array or python list to a python list.
     """
+
+    def _torch_to_py_obj(obj):
+        return obj.detach().cpu().tolist()
+
+    def _tf_to_py_obj(obj):
+        return obj.numpy().tolist()
+
+    def _jax_to_py_obj(obj):
+        return np.asarray(obj).tolist()
+
+    def _np_to_py_obj(obj):
+        return obj.tolist()
+
+    framework_to_py_obj = {
+        "pt": _torch_to_py_obj,
+        "tf": _tf_to_py_obj,
+        "jax": _jax_to_py_obj,
+        "np": _np_to_py_obj,
+    }
```

Comment:
We could used lambda function instead of one liners  ?

Reply:
lambda functions are evil since they don't serialize, so I never use them. But in this instance I don't see this becoming a problem in serialization, so yes, we could.